### ISSUE TITLE
Drug-Drug interactionDisseminated candidiasis

### ISSUE THREAD

**Issue by Marwaabdelhakim** [obophenotype/human-phenotype-ontology#11131]
**Preferred term label:**
Drug-Drug interaction

**Synonyms**
Adverse drug interaction
Contraindicated drug combination

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

Adverse drug response
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
[0000-0001-6816-2119](https://orcid.org/0000-0001-6816-2119)

**Comment by pnrobinson** [comment]
@Marwaabdelhakim I do not think this is an observable phenotype - Contraindicated drug combination is knowledge that the combination of two drugs may lead to side effects. As such, I think that this is outside the scope of the HPO. If you are referring to a phenotypic feature, please open a new term request with a definition and clinical context. Thanks!
